[Graylog] Not starting after fresh install

### ✅ Have you read and understood the above guidelines?

yes

### 🔎 Did you run the script with verbose mode enabled?

Yes, verbose mode was enabled and the output is included below

### 📜 What is the name of the script you are using?

Graylog

### 📂 What was the exact command used to execute the script?

bash -c "$(curl -fsSL https://git.community-scripts.org/community-scripts/ProxmoxVE/raw/branch/main/ct/graylog.sh)"

### ⚙️ What settings are you using?

- [x] Default Settings
- [ ] Advanced Settings

### 🖥️ Which Linux distribution are you using?

Debian 13

### 📈 Which Proxmox version are you on?

9.1.9

### 📝 Provide a clear and concise description of the issue.

Graylog seems to install correctly. In the container if I use `systemctl status graylog-server` or `systemctl status graylog-datanode` it says they are in running state. But if I look into the logs there are several errors.

These are the first lines of server.log
```
root@graylog:~# cat /var/log/graylog-server/server.log 
2026-04-29T13:35:04.105+01:00 INFO  [ImmutableFeatureFlagsCollector] Following feature flags are used: {default properties file=[show_security_events_in_pedt=off, data_tiering_cloud=off, preflight_web=on, configurable_value_units=on, setup_mode=on, cloud_inputs=on, investigation_report_by_ai=on, show_executive_dashboard_page=off, collections=on, composable_index_templates=off, data_node_migration=on, remote_reindex_migration=off, instant_archiving=off, data_lake_search=on, widget_summary=on, threat_coverage=on, external_data_lake_search=on]}
2026-04-29T13:35:04.513+01:00 INFO  [CmdLineTool] Loaded plugin: AWS plugins 7.0.6+711d207 [org.graylog.aws.AWSPlugin]
2026-04-29T13:35:04.514+01:00 INFO  [CmdLineTool] Loaded plugin: Integrations 7.0.6+711d207 [org.graylog.integrations.IntegrationsPlugin]
2026-04-29T13:35:04.514+01:00 INFO  [CmdLineTool] Loaded plugin: Threat Intelligence Plugin 7.0.6+711d207 [org.graylog.plugins.threatintel.ThreatIntelPlugin]
2026-04-29T13:35:04.514+01:00 INFO  [CmdLineTool] Loaded plugin: Elasticsearch 7 Support 7.0.6+711d207 [org.graylog.storage.elasticsearch7.Elasticsearch7Plugin]
2026-04-29T13:35:04.514+01:00 INFO  [CmdLineTool] Loaded plugin: OpenSearch 2 Support 7.0.6+711d207 [org.graylog.storage.opensearch2.OpenSearch2Plugin]
2026-04-29T13:35:04.523+01:00 INFO  [CmdLineTool] Running with JVM arguments: -Xms1g -Xmx1g -XX:+UseG1GC -XX:-OmitStackTraceInFastThrow -Djdk.tls.acknowledgeCloseNotify=true -Dlog4j2.formatMsgNoLookups=true -Dlog4j.configurationFile=file:///etc/graylog/server/log4j2.xml -Dgraylog2.installation_source=deb
2026-04-29T13:35:04.645+01:00 INFO  [client] MongoClient with metadata {"driver": {"name": "mongo-java-driver|legacy", "version": "5.6.1"}, "os": {"type": "Linux", "name": "Linux", "architecture": "amd64", "version": "7.0.0-3-pve"}, "platform": "Java/Eclipse Adoptium/21.0.10+7-LTS"} created with settings MongoClientSettings{readPreference=primary, writeConcern=WriteConcern{w=null, wTimeout=null ms, journal=null}, retryWrites=true, retryReads=true, readConcern=ReadConcern{level=null}, credential=null, transportSettings=null, commandListeners=[], codecRegistry=ProvidersCodecRegistry{codecProviders=[ValueCodecProvider{}, BsonValueCodecProvider{}, DBRefCodecProvider{}, DBObjectCodecProvider{}, DocumentCodecProvider{}, CollectionCodecProvider{}, IterableCodecProvider{}, MapCodecProvider{}, GeoJsonCodecProvider{}, GridFSFileCodecProvider{}, Jsr310CodecProvider{}, JsonObjectCodecProvider{}, BsonCodecProvider{}, com.mo
```

These are the first lines of datanode.log
```
2026-04-29T13:34:57.588+01:00 INFO  [ImmutableFeatureFlagsCollector] Following feature flags are used: {}
2026-04-29T13:34:57.607+01:00 WARN  [PluginLoader] Plugin directory /plugin does not exist, not loading plugins.
2026-04-29T13:34:57.957+01:00 INFO  [CmdLineTool] Running with JVM arguments: -Dlog4j.configurationFile=file:///e>
2026-04-29T13:34:58.314+01:00 INFO  [client] MongoClient with metadata {"driver": {"name": "mongo-java-driver|leg>
2026-04-29T13:34:58.316+01:00 INFO  [client] MongoClient with metadata {"driver": {"name": "mongo-java-driver|leg>
2026-04-29T13:34:58.329+01:00 INFO  [cluster] Exception in monitor thread while connecting to server localhost:27>
com.mongodb.MongoSocketOpenException: Exception opening socket
        at com.mongodb.internal.connection.SocketStream.lambda$open$0(SocketStream.java:85)
        at java.base/java.util.Optional.orElseThrow(Unknown Source)
        at com.mongodb.internal.connection.SocketStream.open(SocketStream.java:85)
        at com.mongodb.internal.connection.InternalStreamConnection.open(InternalStreamConnection.java:233)
        at com.mongodb.internal.connection.DefaultServerMonitor$ServerMonitor.setupNewConnectionAndGetInitialDesc>
        at com.mongodb.internal.connection.DefaultServerMonitor$ServerMonitor.lookupServerDescription(DefaultServ>
        at com.mongodb.internal.connection.DefaultServerMonitor$ServerMonitor.run(DefaultServerMonitor.java:203)
Caused by: java.net.ConnectException: Connection refused
        at java.base/sun.nio.ch.Net.pollConnect(Native Method)
        at java.base/sun.nio.ch.Net.pollConnectNow(Unknown Source)
        at java.base/sun.nio.ch.NioSocketImpl.timedFinishConnect(Unknown Source)
        at java.base/sun.nio.ch.NioSocketImpl.connect(Unknown Source)
        at java.base/java.net.SocksSocketImpl.connect(Unknown Source)
        at java.base/java.net.Socket.connect(Unknown Source)
        at com.mongodb.internal.connection.SocketStreamHelper.initialize(SocketStreamHelper.java:76)
        at com.mongodb.internal.connection.SocketStream.initializeSocket(SocketStream.java:104)
        at com.mongodb.internal.connection.SocketStream.open(SocketStream.java:79
```

### 🔄 Steps to reproduce the issue.

Install with default parameters

### ❌ Paste the full error output (if available).

See above

### 🖼️ Additional context (optional).

_No response_

--- a/ct/graylog.sh
+++ b/ct/graylog.sh
@@ -37,7 +37,7 @@ function update_script() {
   CURRENT_VERSION=$(apt list --installed 2>/dev/null | grep graylog-server | grep -oP '\d+\.\d+\.\d+')
 
   if dpkg --compare-versions "$CURRENT_VERSION" lt "6.3"; then
-    MONGO_VERSION="8.0" setup_mongodb
+    MONGO_VERSION="8.2" setup_mongodb
 
     msg_info "Updating Graylog"
     $STD apt update

--- a/install/graylog-install.sh
+++ b/install/graylog-install.sh
@@ -13,7 +13,7 @@ setting_up_container
 network_check
 update_os
 
-MONGO_VERSION="8.0" setup_mongodb
+MONGO_VERSION="8.2" setup_mongodb
 
 msg_info "Setup Graylog Data Node"
 PASSWORD_SECRET=$(openssl rand -base64 18 | tr -dc 'a-zA-Z0-9' | head -c16)

